packages/sdk-codegen/src/python.gen.spec.ts

```diff
@@ -222,7 +222,7 @@ sr.converter.register_structure_hook(
 # GET /datagroups -> Sequence[models.Datagroup]
 def all_datagroups(
     self,
-    transport_options: Optional[transport.PTransportSettings] = None,
+    transport_options: Optional[transport.TransportOptions] = None,
```

Comment:
oops, typing typo that's been around for a little while

Reply:
I actually saw that one a while back and wondered about it!